util/crypto/crypto.go

```diff
@@ -0,0 +1,53 @@
+package crypto
+
+import (
+	"crypto/aes"
+	"crypto/cipher"
+	"crypto/rand"
+	"crypto/sha256"
+	"io"
+)
+
+func getKey(passphrase string) []byte {
```

Comment:
Maybe rename this function to `passphraseToKey` or `keyFromPassphrase`?

Reply:
Sounds good. Done